app/views/rsvp.erb

```diff
@@ -1,14 +1,34 @@
 <div class="offset" id="rsvp"></div>
+
 <% if session['access_token'] %>
-  <form class="form-horizontal" method="POST" action="/rsvp">
-  <fieldset>
+
+<% # to validate if the user in either of me and Nikhil's friend list
+    graph = Koala::Facebook::API.new(session['access_token'])
+    profile = graph.get_object('me')
+
+    #profile will be a JSON array , retrieve the name from there and display it
+
+    #get the friends list and see what all it displays  
+    friends = graph.get_connections('me','friends')
+    #print friends
+    #print friends.count
+    
+    #also we need to do a check for me and Nikhil, so that we can login and do anything
+    me_or_nikhil = (profile["id"] == ENV["debo_id"] || profile["id"] == ENV["nikhil_id"])?true:false
```

Comment:
+1 Good job with setting the environment variables, we can have one environment variable with a CSV and call it whitelisted_ids. This way we should be able to give access to people who may be in some of our familys friendlists by just changing one variable.